WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -236,6 +246,36 @@ public class BlockEditorScreen: ScreenObject {
         blockButton.tap()
     }
 
+    @discardableResult
+    public func undo() throws -> BlockEditorScreen {
+        XCTAssertTrue(undoButton.waitForIsHittable(timeout: 3))
+        undoButton.tap()
+
+        return try BlockEditorScreen()
+    }
+
+    @discardableResult
+    public func undoIsDisabled() throws -> BlockEditorScreen {
```

Comment:
```suggestion
    public func verifyUndoIsDisabled() throws -> BlockEditorScreen {
```
This would be just to follow the existing pattern to differentiate action steps from assertions, making it clearer for someone reading the test case.

Reply:
Sounds good! I've updated it in https://github.com/wordpress-mobile/WordPress-iOS/pull/20930/commits/9747901ba9f5465abdb836197773818f4988ec31 thanks!